flow-server/src/main/java/com/vaadin/flow/server/communication/UidlRequestHandler.java

```
@@ -102,48 +103,64 @@ protected ServerRpcHandler createRpcHandler() {
     @Override
     public boolean synchronizedHandleRequest(VaadinSession session,
             VaadinRequest request, VaadinResponse response) throws IOException {
+        String requestBody = SynchronizedRequestHandler
+                .getRequestBody(request.getReader());
+        Optional<ResponseWriter> responseWriter = synchronizedHandleRequest(
+                session, request, response, requestBody);
+        if (responseWriter.isPresent()) {
+            responseWriter.get().writeResponse();
+        }
+        return responseWriter.isPresent();
```

Why is needed to override this `synchronizedHandleRequest(...)` method? Isn't this logic already applied in `SynchronizedRequestHandler` if `isReadRequestBodyFirstEnabled()` is true?

---

Well that comes from `RequestHandler` interface so I must implement something. Would it be better to just throw `UnsupportedOperationException` since this should never be called for this handler?